src/accounts.rs

```diff
@@ -271,14 +271,14 @@ impl Accounts {
     /// Notifies all accounts that the network may have become available.
     pub async fn maybe_network(&self) {
         for account in self.accounts.values() {
-            account.maybe_network().await;
+            account.scheduler.maybe_network().await;
```

Comment:
I've moved these methods to be on the scheduler now that the scheduler can't be None.  I think this makes more sense and keeps things more structured.  It does generate a little extra noise though.